apache-rat-tasks/src/test/java/org/apache/rat/anttasks/ReportTest.java

```diff
@@ -260,4 +261,9 @@ public void testISO88591WithFile() throws Exception {
         }
         assertTrue("Report file could not be parsed as XML", documentParsed);
     }
+
+    public void testReportOptions(String antFileContents, Predicate<ReportConfiguration> test) {
+
```

Comment:
Is there a testcase missing here or does Ant not support this part of the config? just wondering